usaspending_api/api_contracts/contracts/v2/download/count.md

```diff
@@ -19,6 +19,11 @@ Returns the number of transactions that would be included in a download request
 
     + Attributes (object)
         + `filters` (required, AdvancedFilterObject)
+        + `spending_level` (optional, enum[string])
```

Comment:
Can you also add the `Default` here.

Example https://github.com/fedspendingtransparency/usaspending-api/blob/bb493c82f36798f07e1ead75ec9d0c2a5c293ffe/usaspending_api/api_contracts/contracts/v2/subawards.md?plain=1#L15